### PR TITLE
feature/AT-RSC-UI-Teams

### PR DESCRIPTION
**Pull request by acones96**
feature/AT-RSC-UI-Teams

### CODE REVIEW

**Comment by at-internship** [comment]
Please add @at-internship & @raaul-at as a reviewers

File: src/views/teams/team-list.hbs [hunk 1]
```diff
@@ -0,0 +1 @@
+Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque molestiae rem alias aut dicta numquam labore, molestias dolorum. Amet ut pariatur possimus. Ipsam ad nobis sint provident, architecto consequuntur omnis.
```
Comment (at-internship) [inline src/views/teams/team-list.hbs]: 1.- use this path `src/views/teams.hbs`
2.- Add an "empty" hbs like:
```
<div class="container-fluid">
    <div class="row">
        <nav id="sidebarMenu" class="col-md-3 col-lg-2 d-md-block bg-light sidebar collapse">
            <div class="sidebar-sticky pt-3">
                <!-- MENU-AT-RESOURCES -->
                {{> menu-at-resources }}
                <!-- MENU-AT-RESOURCES -->
            </div>
        </nav>

        <!-- TEAMS -->
        <main role="main" class="col-md-9 ml-sm-auto col-lg-10 px-md-5">
            <div
                class="d-flex justify-content-between flex-wrap flex-md-nowrap align-items-center pt-3 pb-2 mb-3 border-bottom">
                <h1 class="h2">TEAMS</h1>
            </div>
        </main>
        <!-- TEAMS -->
    </div>
</div>
```

**Review by at-internship — CHANGES_REQUESTED** [review]

File: src/views/teams/team-list.hbs (hunk 1 above)
Comment (acones96) [reply src/views/teams/team-list.hbs]: Got it

**Review by raaul-at — APPROVED** [review]
👍

**Review by at-internship — APPROVED** [review]
Approved

**Pull request merged**